baker/algolia/utils/mdimViews.ts

```diff
@@ -81,19 +81,22 @@ async function getRecords(
         const queryStr = queryParamsToStr(view.dimensions)
         const variableId = view.indicators.y[0].id
         const metadata = _.merge(
+            {}, // merge mutates the first argument
             relevantVariableMetadata[variableId],
             multiDim.config.metadata,
             view.metadata
         )
         const title = maybeAddChangeInPrefix(
-            metadata.presentation?.titlePublic ||
-                chartConfig.title ||
+            chartConfig.title ||
+                metadata.presentation?.titlePublic ||
```

Comment:
The chart config's title should trump the metadata, no?

Reply:
Hmm, I now also looked at what config we pass into `getDatapageDataV2` in the normal data page code path, and it's either the page chart config or the page chart config merged with the indicator chart config:

https://github.com/owid/owid-grapher/blob/da854aab83438c13fb813478ad4bd3d69decd51e/baker/GrapherBaker.tsx#L155-L157

So that would correspond to the mdim view config (which is a merge of the indicator config and mdim config override), wouldn't it?

Reply:
mhh, not sure to be honest. leaving as is then